packages/vkui/src/components/Clickable/Clickable.tsx

```diff
@@ -74,7 +74,7 @@ const RealClickable = <T,>({
     activated,
   });
 
-  const keyboardHandlers = useKeyboard();
+  const keyboardHandlers = useKeyboardClick();
 
   const handlers = mergeCalls(focusEvents, stateEvents, keyboardHandlers, {
```

Comment:
```suggestion
  const handlers = mergeCalls(focusEvents, stateEvents, { onKeyDown: clickByKeyboardHandler }, {
```

заметил, что хук нам не нужен, достаточно объявить функцию `clickByKeyboardHandler`